lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
      *
      * @param HydrateEvent $event
      */
-    public function handleDefaultLocale(HydrateEvent $event)
+    public function handleDefaultLocale(AbstractMappingEvent $event)
```

Comment:
The fact that we use [checks like this](https://github.com/sulu/sulu/blob/7f745ac7c5617bfcf3a92de53871e03829c08a19/src/Sulu/Component/Content/Document/Subscriber/StructureSubscriber.php#L202) makes this change of the document manager incompatible with the current state of Sulu. Also, it still doesn't make any sense to pass a locale when we e.g. move a page, since this change is valid for all localizations, and not only a single one. This is just confusing and screams for unexpected behavior.